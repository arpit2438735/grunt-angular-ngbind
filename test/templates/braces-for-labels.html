<label>
    {{labelValue}}
    <input type="text">
</label>