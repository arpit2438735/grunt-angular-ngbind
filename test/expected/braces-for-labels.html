<label><span ng-bind="labelValue"></span><input type="text"></label>